lib/addRemove.js

```diff
@@ -64,7 +64,7 @@ function addAsync(version, useNow) {
 			let versionDir = nvsUse.getVersionDir(version);
```

Comment:
I wanted to, but that would have shown lot of diffs. 😄 . Curious, why you choose tabs over spaces?